plugins/chat/assets/javascripts/discourse/models/chat-thread.js

```diff
@@ -81,6 +81,10 @@ export default class ChatThread {
     return this.messagesManager.findLastUserMessage(user);
   }
 
+  clearSelectedMessages() {
```

Comment:
Could we add this for the `chat-channel` model too? I see we are still doing `message.selected = false` manually in `plugins/chat/assets/javascripts/discourse/services/chat-channel-pane.js`

Reply:
yes sure :+1: